omnibus/resources/agent/msi/cal/CustomAction.cpp

```diff
@@ -40,162 +32,16 @@ extern "C" UINT __stdcall FinalizeInstall(MSIHANDLE hInstall) {
 
     // check to see if we're a domain controller.
     WcaLog(LOGMSG_STANDARD, "checking if this is a domain controller");
-    isDC = isDomainController(hInstall);
 
-    // check to see if the supplied dd-agent-user exists
-    WcaLog(LOGMSG_STANDARD, "checking to see if the user is already present");
-    if ((ddUserExists = doesUserExist(hInstall, data, isDC)) == -1) {
-        er = ERROR_INSTALL_FAILURE;
-        goto LExit;
-    }
-    // check to see if the service is already installed
-    WcaLog(LOGMSG_STANDARD, "checking to see if the service is installed");
-    if ((ddServiceExists = doesServiceExist(hInstall, agentService)) == -1) {
-        er = ERROR_INSTALL_FAILURE;
-        goto LExit;
-    }
     
-    // now we have all the information we need to decide if this is a
-    // new installation or an upgrade, and what steps need to be taken
-
-    ///////////////////////////////////////////////////////////////////////////
-    //
-    // If domain controller:
-    //   If user is present:
-    //     if service is present:
-    //        this is an upgrade.
-    //     if service is not present
-    //        this is new install on this machine
-    //        dd user has already been created in domain
-    //        must have password for registering service
-    //   If user is NOT present
-    //     if service is present
-    //       ERROR how could service be present but user not present?
-    //     if service is not present
-    //       new install in this domain
-    //       must have password for user creation and service installation
-    //
-    // If NOT a domain controller
-    //   if user is present
-    //     if the service is present
```

Comment:
Note: The following functions are not called anymore:
 - generatePassword
 - doesUserExist
 - setUserProfileFolder
 - getUserProfileFolder
 - AddPrivileges